src/gamecontroller.cs

```diff
@@ -2018,7 +2042,7 @@ interface GCVirtualController
 		[NullAllowed, Export ("controller", ArgumentSemantic.Weak)]
 		GCController Controller { get; }
 
-		[Export ("changeElement:configuration:")]
+		[Export ("updateConfigurationForElement:configuration:")]
 		void Change (string element, GCVirtualControllerElementUpdateBlock configuration);
```

Comment:
Maybe rename the managed method too?

```suggestion
		void UpdateConfiguration (string element, GCVirtualControllerElementUpdateBlock configuration);
```

Reply:
in doubt check the swift name :)